File: Programmers/Java/Lv.0/[Lv.0] 외계행성의 나이/[Lv.0] 외계행성의 나이_00.java
class Solution {
    public String solution(int age) {
        String answer = "";
        String planet = String.valueOf(age);
        String[] str = planet.split("");

        for (int i = 0; i < planet.length(); i++) {
            // + 97 : 아스키 코드 변환
            // char 문자 변환
            answer += (char) (Integer.parseInt(str[i]) + 97);
        }

        return answer;
    }
}

File: Programmers/Python/Lv.0/[Lv.0] 외계행성의 나이/[Lv.0] 외계행성의 나이_00.py
def solution(age):
    answer = ''

    for i in str(age):
        # 97, 'a' : 아스키 코드 변환
        # chr() : 아스키 코드 -> 문자열 변환
        # ord() : 문자열 -> 아스키 코드 변환
        answer += chr(int(i) + 97)
        # answer += chr(ord('a') + int(i))

    return answer
